pkg/dns/dns.go

```diff
@@ -51,6 +52,9 @@ const (
 	// A subdomain added to the user specified dmoain for all pods.
 	podSubdomain = "pod"
 
+	// A subdomain added to the user specified domain for all ingresses.
+	ingressSubdomain = "ing"
```

Comment:
@thockin @bprashanth Any objections to this?  It's the equivalent to '.svc' for services. What process do we need to follow to take this out of Alpha?  Is it a big enough decision to drag through the networking/API SIG's?  I suspect not.